aptos-move/move-examples/event/sources/event.move

```diff
@@ -0,0 +1,54 @@
+/// This provides an example shows how to use module events.
+
+module event::event {
+    use aptos_framework::event;
+    #[test_only]
+    use std::vector;
+
+    struct Field has store, drop {
+        field: bool,
+    }
+
+    #[event]
+    struct MyEvent has store, drop {
```

Comment:
What happens if you have weird data structures in here such as Table?

Reply:
I think the handle will be serialized.